<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageTaskScriptingWidget</class>
 <widget class="QWidget" name="ImageTaskScriptingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QPushButton" name="editShapeScriptsButton">
     <property name="text">
      <string>Edit Shape Creation/Mutation Scripts</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="customScriptsGroupBox">
     <property name="title">
      <string>Custom Scripts</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="addBeforeStepEvent">
            <property name="text">
             <string>Add Before Step Event</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="addBeforeStepPresetScriptsComboBox">
            <property name="currentText">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="addAfterStepEvent">
            <property name="text">
             <string>Add After Step Event</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="addAfterStepPresetScriptsComboBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="addStopConditionButton">
            <property name="text">
             <string>Add Stop Condition</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="addStopConditionPresetScriptsComboBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="addBeforeAddShapeEvent">
            <property name="text">
             <string>Add Before Add Shape Event</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="addBeforeAddShapePresetScriptsComboBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="addAfterAddShapeEvent">
            <property name="text">
             <string>Add After Add Shape Event</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="addAfterAddShapePresetScriptsComboBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="customScriptsEditorLayout"/>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rescanScriptsButton">
        <property name="text">
         <string>Rescan Scripts</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearScriptsButton">
        <property name="text">
         <string>Clear Scripts</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
